app/models/ErrorResponse.scala

```diff
@@ -48,4 +48,6 @@ case object GatewayTimeout extends ErrorResponse {
   override val body: String = "Gateway timeout"
 }
 
+case class ETMPException(message: String) extends Exception(message)
```

Comment:
Should be camel case i.e. "EtmpException" - https://docs.scala-lang.org/style/naming-conventions.html